src/SceneMap.js

```diff
@@ -2,10 +2,24 @@
 
 import * as React from 'react';
 
-class SceneComponent extends React.PureComponent<*> {
+type State = {|
+  shouldRenderContent: boolean,
+|};
+
+class SceneComponent extends React.PureComponent<*, State> {
+  constructor(props) {
+    super(props);
+    this.state = { shouldRenderContent: false };
+    //delay rendering of not focused tabs to the next tick:
+    setTimeout(() => this.setState({ shouldRenderContent: true }), 0);
```

Comment:
Can you move this code to `TabView`? This can be done for all scenes, not only the ones using `SceneMap`

Reply:
@Niryo you can add a variable to state indicating if it's initial render, and then add a check like `this.state.initial || i === navigationState.index` here: https://github.com/react-native-community/react-native-tab-view/blob/master/src/TabView.js#L193